src/test/kotlin/com/igorwojda/string/ispalindrome/permutation/challenge.kt

```diff
@@ -4,7 +4,16 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun isPermutationPalindrome(str: String): Boolean {
-    TODO("not implemented")
+    var oneCharAllowed = false
+    str.forEach {
+        val noChars = str.filter { c ->
```

Comment:
Please make variable names more readable, e.g. noChars can be charOccurrences or charCount.